modules/time.js

```diff
@@ -0,0 +1,11 @@
+/* eslint-disable linebreak-style */
```

Comment:
- [ ] Please remove the code to disable the linter and fix linter errors manually, It will help you to write clean code.

Reply:
for this error to be solved we must make changes on the code in  eslintrc file which we are instructed not to